src/olympia/devhub/templates/devhub/addons/submit/done.html

```diff
@@ -57,6 +57,8 @@ <h3>{{ _("Version Signature Pending") }}</h3>
   <p>
     {{ _("You’re done! ✨ You will be notified by email when the signed file is ready to be downloaded from the Developer Hub. If you do not see an email after 24 hours, please check your spam folder.") }}
   </p>
+{% endif %}
+{% if addon.type != amo.ADDON_STATICTHEME %}
```

Comment:
I'm not sure it's worthwhile specifically excluding themes from the statement (presumably they could also be manual reviewed under some circumstances)

Reply:
Themes are manually approved in _all_ circumstances - it's much less likely that a new review would change the status and that the author would be surprised about it.